packages/api-bindings/src/apollo/createExploreProfileFieldPolicy.ts

```diff
@@ -1,5 +1,7 @@
 import { cursorBasedPagination } from './utils/cursorBasedPagination';
 
 export function createExploreProfilesFieldPolicy() {
-  return cursorBasedPagination([['request', ['observerId']]]);
+  return cursorBasedPagination([
+    ['request', ['sortCriteria', 'timestamp', 'publicationTypes', 'excludeProfileIds']],
```

Comment:
so the changes to add these to the `useExploreProfiles` allowed args gonna be done later, right?

Reply:
I think these are already there for `useExploreProfiles`, I checked the hook args to come up w/ this list.

Fixed it here cause I stumbled on this misconfiguration and it was a quick fix.